modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewPage.kt

```diff
@@ -130,12 +135,7 @@ private fun WhatsNewPageLoaded(
                 if (state.fullModel) {
                     Spacer(modifier = Modifier.weight(0.2f))
                 }
-                Box(
-                    contentAlignment = Alignment.Center,
-                    modifier = Modifier.weight(1f),
-                ) {
-                    header()
-                }
+                header()
```

Comment:
I removed the box back in order to put the tittle in center as we were doing before sharing clip whats new